packages/wonder-blocks-banner/package.json

```diff
@@ -21,6 +21,7 @@
     "@khanacademy/wonder-blocks-core": "^4.4.0",
     "@khanacademy/wonder-blocks-icon": "^1.2.31",
     "@khanacademy/wonder-blocks-icon-button": "^3.4.12",
+    "@khanacademy/wonder-blocks-link": "^3.8.10",
```

Comment:
**suggestion:** It looks like the latest version is `3.8.12`:

For reference: https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-link/package.json#L3

```suggestion
    "@khanacademy/wonder-blocks-link": "^3.8.12",
```